### PR TITLE
Link Action - Plain Embed Rendering Update

### PR DESCRIPTION
**Pull request by mendelB**
### What's this PR do?

This pull request updates the `LinkAction` (`DefaultTemplate`) to no longer support rendering as a plain 'embed' link. We'll still support any Link Actions created before this deprecation.

It also merges in @Dinnall's work in #2478 to enable analytics tracking for link clicks on links rendered from the `Embed`, along with a `noTrack` prop to enable skipping this duplicitous tracking on legacy Link Actions rendered as plain embeds per https://github.com/DoSomething/phoenix-next/pull/1598#issuecomment-532744108

### How should this be reviewed?
👀 

- Do you agree with this #hack?
- Does the `noTrack` prop make sense?

### Any background context you want to provide?
We'd like to start rendering plain embed links (as well as Youtube videos et al) with the **Embed** Contentful block instead of hacking this via the Link Action. Link Action blocks should - and will from this point - always render as a button link.

I'll be following up on this PR with updates to the Link Action content type in Contentful to make this clear to editors, as well as some documentation on the Link Action.

### Relevant tickets

References [Pivotal #175380207](https://www.pivotaltracker.com/story/show/175380207).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added screenshots of front-end changes on small, medium, and large screens.
- [x] Added appropriate feature/unit tests.

### CODE REVIEW

File: resources/assets/components/actions/LinkAction/templates/DefaultTemplate.js [hunk 1]
```diff
@@ -48,15 +50,18 @@ const DefaultTemplate = props => {
   const context = { blockId: id, campaignId, pageId };
 
   // If no content is provided, show as an embed.
-  if (!content) {
+  // HACK: We've deprecated this functionality in favor of using the Embed Block.
+  // This will now only apply to Link Actions created before 01/22/2021.
+  if (!content && isBefore(createdAt, new Date('01/22/2021'))) {
```
Comment (aaronschachter) [inline resources/assets/components/actions/LinkAction/templates/DefaultTemplate.js:55]: Is there a plan to backfill these ever?

**Review by aaronschachter — APPROVED** [review]
Looks good to me. I think `dontTrack` or `noTracking` might be a little more readable than `noTrack`, but not a biggie.

File: resources/assets/components/actions/LinkAction/templates/DefaultTemplate.js (hunk 1 above)
Comment (mendelB) [reply resources/assets/components/actions/LinkAction/templates/DefaultTemplate.js:55]: Not actively as of now, but perhaps in a later cleanup sprint! I'll add a `TODO` here for backfilling this.

**Comment by mendelB** [comment]
> Looks good to me. I think `dontTrack` or `noTracking` might be a little more readable than `noTrack`, but not a biggie.

Yeah I like `dontTrack`! I'll update.

**Comment by mendelB** [comment]
Closes #2478 cc @Dinnall

**Pull request merged**